src/app/pages/PerpetualPage/components/ToastsWatcher/components/ToastAdditionalInfo.tsx/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { AssetValue } from 'app/components/AssetValue';
+import { AssetValueMode } from 'app/components/AssetValue/types';
+import { ToastTransaction } from 'app/pages/PerpetualPage/contexts/ToastsContext';
+import { translations } from 'locales/i18n';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { TradingPosition } from 'types/trading-position';
+import { weiToNumberFormat } from 'utils/display-text/format';
+import {
+  isTrade,
+  isDepositMargin,
+  isWithdrawMargin,
+} from '../../../TradeDialog/types';
+
+type ToastAdditionalInfoProps = {
+  transaction: ToastTransaction;
+};
+
+export const ToastAdditionalInfo: React.FC<ToastAdditionalInfoProps> = ({
+  transaction,
+}) => {
+  const { t } = useTranslation();
+
+  if (isTrade(transaction)) {
+    const amount = weiToNumberFormat(transaction.amount, 3);
+    if (transaction.isClosePosition) {
+      return (
+        <>
+          {t(translations.perpetualPage.toasts.closePosition)} {amount} BTC
```

Comment:
Should not be BTC but baseAsset, you may need to add the pair to the PerpetualTx type, and use it here.